CHANGES

```diff
@@ -94,10 +96,10 @@
     * Removing command on initial connections (#1722)
     * Removing hiredis warning when not installed (#1721)
 * 4.0.0 (Nov 15, 2021)
-    * FT.EXPLAINCLI intentionally raising NotImplementedError 
+    * FT.EXPLAINCLI intentionally raising NotImplementedError
```

Comment:
sorry for these whitespace trims -- my IDE does it 😅

Reply:
flake8 actually enforces it, but only on python files ;)